src/components/User/Users/components/UserList.vue

```diff
@@ -197,6 +191,21 @@ export default {
       this.isLoaded = true
     },
 
+    async setFilters () {
+      if (this.specificUserAddress) {
+        const idLength = 56
```

Comment:
in the current file just the same issues you got in `src/components/User/KycRequests/KycRequest.List.vue`